lib/PerlIO.pm

```diff
@@ -35,14 +35,19 @@ PerlIO - On demand loader for PerlIO layers and root of PerlIO::* name space
 
 =head1 SYNOPSIS
 
-  open($fh, "<:crlf", "my.txt"); # support platform-native and 
-                                 # CRLF text files
+  # support platform-native and CRLF text files
+  open(my $fh, "<:crlf", "my.txt") or die "open failed: $!";
 
-  open($fh, "<", "his.jpg"); # portably open a binary file for reading
-  binmode($fh);
+  # append UTF-8 encoded text
+  open(my $fh, ">>:encoding(UTF-8)", "some.log")
+    or die "open failed: $!";
+
+  # portably open a binary file for reading
```

Comment:
Why not just go straight for `:raw`?

Reply:
binmode $fh will work consistently before Perl 5.14 and on windows as explained later